### PR TITLE
`Locations` command

### PR DESCRIPTION
**Pull request by vizvamitra**
I've implemented `Locations` command which provides user with conference/afterparty description and geo position. Note that I've removed `Places` command from main keyboard. 

[Video](https://drive.google.com/open?id=1v4EieufAtFZ3_J4K4E3dD5bxfx4F7h_M)

  

Also there is an issue with Google maps (don't know about Apple) not knowing about JetBrains' new building: it is present on a map preview but absent when the location is being opened in a Google maps app (see screenshots below). This is why I've added this `(The building is newly built so it may not show up on your map)` string

### CODE REVIEW

File: app/commands/start.rb [hunk 1]
```diff
@@ -3,7 +3,8 @@ class Start < Base
     MENU_KEYBOARD = reply_keyboard(
       [ button('📆 Schedule'), button('❤️ Vote') ],
       [ button('🎤 Speakers'), button('💵 Jobs') ],
-      [ button('🍻 Drunk beer!'), button('🏛 Places') ]
+      [ button('🍻 Drunk beer!'), button('📌 Locations') ],
+      # [ button('🏛 Places') ],
```
Comment (we138) [inline app/commands/start.rb]: does it need?

File: app/dispatcher.rb [hunk 2]
```diff
@@ -9,7 +9,8 @@ def initialize(bot)
       '🎤 Speakers' => Commands::Speakers.new(bot.api),
       '💵 Jobs' => Commands::Jobs.new(bot.api),
       '🍻 Drunk beer!' => Commands::Beers.new(bot.api),
-      '🏛 Places' => Commands::Places.new(bot.api),
+      '📌 Locations' => Commands::Locations.new(bot.api),
+      # '🏛 Places' => Commands::Places.new(bot.api),
```
Comment (we138) [inline app/dispatcher.rb]: does it need?

Comment (vizvamitra) [reply app/dispatcher.rb]: Not sure)

**Review by we138 — APPROVED** [review]

**Comment by we138** [comment]
looks good for me

**Pull request merged**